### DISCUSSION TITLE
Minimum time for a fan to start up

### DISCUSSION THREAD

**Discussion by julienl123** [Rem0o/FanControl.Releases#716] [Ideas]
Hi, thank you very much for this incredible piece of software! A worthy legacy to Speedfan ;)

Would it be possible to add a feature ensuring a minimum duration between a fan waking up from 0 rpm and going back down to 0 rpm?

Some times, fans can go from 0% to 30% (because not possible to be controlled between the first 30%) and then the temperatures goes down quickly and the fan shuts off. It can be quite disturbing.

Let me know ;)

and thanks again!!

**Comment by Fr0stX76** [comment]
Hysteresis in graph is exactly what this is for. Inclease the response time and the hysteresis temperature and this will be solved

I would suggest to starting with 5 seconds and 2 degree C and go from there. Also, using the only apply on the way down to make sure it does not add any delay to the fan ramp up on temp increase (only on temp decrease)

**Reply by julienl123** [reply]
thanks for the reply but I don't think it quite fit my need.

Quick question: does the response time works on decrease only if I check the box or is only for hysteresis?
